plugins/coverbrowser.koplugin/mosaicmenu.lua

```diff
@@ -136,6 +137,16 @@ function FakeCover:init()
         -- Also replace underscores with spaces
         title = title:gsub("_", " ")
     end
+    -- If multiple authors (crengine separates them with \n), we
+    -- can display them on multiple lines, but limit to 3, and
+    -- append "et al." on a 4th line if there are more
+    if authors and authors:find("\n") then
+        authors = util.splitToArray(authors, "\n")
+        if #authors > 3 then
+            authors = { authors[1], authors[2], authors[3], _("et al.") }
```

Comment:
Probably better as `T(_("%1 et al."), authors[3])`?

Reply:
OK, less room wasted = bigger font possible.

Reply:
If it's all by itself I think it should be `et alia`, which as you already indicated is kind of weird in a way. :-P